packages/web/src/capture/styles.css

```
@@ -28,28 +28,29 @@
-  --ink: #ece8e1;
-  --ink-soft: #b4afa5;
-  --ink-faint: #8b867c;
-  --line: #37342f;
+  --bg: #002b36;
+  --surface: #073642;
+  --seg: #0d4152;
+  --ink: #93a1a1;
+  --ink-soft: #7b8e93;
+  --ink-faint: #657b83;
+  --line: #1a4a5a;
   color-scheme: dark;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)